cmd/kubeadm/app/cmd/init.go

```diff
@@ -65,22 +65,27 @@ func NewCmdInit(out io.Writer, s *kubeadmapi.KubeadmConfig) *cobra.Command {
 		&s.InitFlags.API.ExternalDNSNames, "api-external-dns-names", []string{},
 		`(optional) The DNS names to advertise, in case you have configured them yourself.`,
 	)
-
 	cmd.PersistentFlags().IPNetVar(
 		&s.InitFlags.Services.CIDR, "service-cidr", *kubeadmapi.DefaultServicesCIDR,
-		`(optional) use alternative range of IP address for service VIPs, e.g. "10.16.0.0/12"`,
+		`(optional) Use alterantive range of IP address for service VIPs, defaults to `+
+			kubeadmapi.DefaultServicesCIDRString,
+	)
+	cmd.PersistentFlags().IPNetVar(
+		&s.InitFlags.PodNetwork.CIDR, "cluster-cidr", net.IPNet{},
```

Comment:
I think the flag should be `--pod-cidr`, because `--cluster-ip-range` in `kube-apiserver` means what we have as `--service-cidr`. I think pods and service are a better grouping, and the exact flags `kube-apiserver` takes aren't as clear, but we shouldn't conflate things even further. We are likely going to revisit the flags throughout alpha phase, so it's not set in stone yet, but please change this to `--pod-cidr`.